Cargo.toml

```diff
@@ -14,9 +14,12 @@ base64 = "0.13.0"
 biscuit-auth = { version = "3.0.0-alpha2", features = ["wasm", "serde-error"] }
 biscuit-parser = { version = "0.1.0-alpha2", features = ["serde-error"] }
 console_error_panic_hook = "0.1.7"
+getrandom = { version = "0.2.8", features = ["js"] }
```

Comment:
this should probably be done in the biscuit-auth crate

Reply:
oh, another thing here: rand and getrandom cannot be updated yet, because biscuit uses [crypto crates](https://github.com/dalek-cryptography/ed25519-dalek) that still rely on old rand versions. Maintenance has restarted on those crates so I have good hopes it will be fixed soon and ship in the next biscuit update

Reply:
(there's a trick I could test where I externaliser random number generation to the biscuit-wasm library, that could be enough to fix it until the dalek crate is updated)